### ISSUE TITLE
Add data examples and create tables 

### ISSUE THREAD

**Issue by zhenik** [fredrikhgrelland/terraform-nomad-presto#5]
- [x] csv
- [x] json
- [x] avro
- [ ] protobuf

**Comment by zhenik** [comment]
CSV support blocker https://github.com/fredrikhgrelland/terraform-nomad-hive/issues/12 

References: https://towardsdatascience.com/load-and-query-csv-file-in-s3-with-presto-b0d50bc773c9

> Presto’s CSV format support requires metastore.storage.schema.reader.impl=org.apache.hadoop.hive.metastore.SerDeStorageSchemaReader in the metastore-site.xml Hive Metastore configuration file.

## How to resolve
- [ ] fix blocker
- [ ] add additional env var to hive, when deploy

```hcl
hive_container_environment_variables =[ 
 "HIVE_SITE_CONF_metastore_storage_schema_reader_impl=org.apache.hadoop.hive.metastore.SerDeStorageSchemaReader"
]
```

**Comment by zhenik** [comment]
## Update 2

[presto-cli](https://prestosql.io/docs/current/installation/cli.html) require java.
Blocker on `vagrant-hashistack` [#304](https://github.com/fredrikhgrelland/vagrant-hashistack/issues/304)

## Log
```text
vagrant@vagrant:/vagrant/dev/ansible$ ../../presto_340 --server localhost:8080 --catalog hive --schema default --user presto --file ../../example/resources/query/csv_create_table.sql
../../presto_340: 3: exec: java: not found
vagrant@vagrant:/vagrant/dev/ansible$ which java
vagrant@vagrant:/vagrant/dev/ansible$ echo $JAVA_HOME

vagrant@vagrant:/vagrant/dev/ansible$ 
```

## How to resolve
- [  ] fix blocker [#304](https://github.com/fredrikhgrelland/vagrant-hashistack/issues/304)
